cooltools/cli/compute_expected.py

```diff
@@ -51,14 +51,23 @@
 @click.option(
     "--contact-type",
     "-t",
-    help="compute expected for cis or trans region of a Hi-C map.",
+    help="compute expected for cis or trans region of a Hi-C map."
+         "Ignored when genomic-regions are provided",
     type=click.Choice(["cis", "trans"]),
     default="cis",
     show_default=True,
 )
+@click.option(
+    "--genomic-regions",
```

Comment:
this is just a placeholder for genomic regions as a bed/bedpe file ...
but I actually don't like it - i think it would make code really complicated - esp. inferring type of expected for every region or a pair of regions ...
the original idea in #127 
> of course we'd need to introduce --range (and potentially --range2) to calculate intra-armal expected (which might be useful for dot calling btw)

seems much more feasible flexible and nice ...

Although it would require ~40 CLI calls to calculate cis-expected per-arm ...

Not sure how to strike a balance between flexibility, practicality and simplicity of implementation
@nvictus @golobor  what do you think ?

Reply:
Tentative resolution: Call it `--regions`. It will take a BED4-style file (chrom, start, end, name) so that each region gets a unique name. Keep as placeholder for now.